native/packages/react-native-bpk-component-button/stories.js

```diff
@@ -215,6 +222,14 @@ storiesOf('BpkButton', module)
       {generateButtonStoryForType('featured')}
     </View>
   ))
+  .add('docs:withTheme', () => (
+    <BpkThemeProvider theme={themeAttributes}>
+      <ScrollView>
+        <BpkText textStyle="xxl">Primary</BpkText>
+        {generateButtonStoryForType('primary')}
+      </ScrollView>
+    </BpkThemeProvider>
+  ))
```

Comment:
Notice that it now uses the same method as other button types - the only difference is it's wrapped in `BpkThemeProvider` 😄

Reply:
👍